utils/atomsClassnameTableData.ts

```diff
@@ -0,0 +1,40 @@
+type AtomsClassnamesProp = {
+    name: string;
+    description: string
+}
+
+export const googleCalendarAtomClassnamesPropData: AtomsClassnamesProp[] = [
+    {name: 'className', description: 'Pass in custom classnames from outside for styling the button'},
+]
+
+export const availabilitySettingsAtomClassnamesPropData: AtomsClassnamesProp[] = [
```

Comment:
Question about availability settings and booker atom props - google calendar atom allows "className" prop, but availability settings and booker does not?